packages/app-builder-lib/templates/nsis/uninstaller.nsh

```diff
@@ -131,7 +131,12 @@ Function un.restoreFiles
     Exch $R0
 FunctionEnd
 
-Section "un.install"
+!ifndef UNINSTALL_SECTION_NAME
+  !define UNINSTALL_SECTION_NAME "Uninstall"
```

Comment:
Is this correct? Previously the section was `un.install`.

Reply:
Yes.

First argument in Section is "name", if have prefix "un.", when is section for uninstaller.

I need rename this uninstall section for user (like screenshot in description) "Odinstalovat prohlížeč".

If someone only add some more uninstall section, when default value will be "Uninstall", it is more better than previous "install" :).

I hope that makes sense.

Reply:
Makes sense. I guess I'm more so wondering then, is this a backward compatible change or a breaking change for existing scripts?

Reply:
No breaking change for existing scripts, by default, this section is invisible.